test/backend/inference_backend.py

```diff
@@ -239,7 +239,17 @@ def get_test_models():
 
         # ConvInteger
 
+        # ==OP== ConvTranspose
         # ConvTranspose
+        "test_convtranspose_1d_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{1}},
+        "test_convtranspose_3d_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{1}},
+        # "test_convtranspose_autopad_same_cpu": {STATIC_SHAPE:{}, DYNAMIC_SHAPE:{-1:{-1}}, CONSTANT_INPUT:{1}},
```

Comment:
Please add a comment saying why this is not supported, and perhaps adding the same comment to L242 so that the generated document has it.

Reply:
Done. Thanks.